menas/src/main/scala/za/co/absa/enceladus/menas/controllers/RunController.scala

```diff
@@ -72,6 +72,12 @@ class RunController @Autowired()(runService: RunService) extends BaseController
     runService.getRunSummariesPerDatasetVersion(datasetName)
   }
 
+  @GetMapping(Array("/bySparkAppId/{appId}"))
+  @ResponseStatus(HttpStatus.OK)
+  def getRunBySparkAppId(@PathVariable appId: String): CompletableFuture[String] = {
```

Comment:
Could you write some tests for this in the `RunApiIntegrationSuite`?

Reply:
Added in 1119d36, thnx